RELEASE_NOTES.rst

```diff
@@ -1,3 +1,39 @@
+coala-bears 0.10.0
+==================
+
+<!-- upto 1096b293af15059ce363c0569d5927de2cefd9c2 -->
+**Bug Fixes** 
+
+- `InvalidLinkBear` now extracts links from markdown correctly. [Issue 1338](https://github.com/coala/coala-bears/issues/1338)
```

Comment:
put the issue mention on a new line please, it looks visually cleaner int he code.